packages/core-js-compat/src/data.js

```diff
@@ -1506,19 +1506,26 @@ const data = {
     safari: '1.0',
   },
   'web.url': {
-    chrome: '67',
-    firefox: '57',
+    chrome: '51',
+    edge: '13',
+    firefox: '54',
     node: '10.0',
+    safari: '10.0'
   },
   'web.url.to-json': {
     chrome: '71',
-    firefox: '57',
+    edge: '17',
+    firefox: '54',
     node: '10.0',
+    safari: '10.0'
   },
   'web.url-search-params': {
-    chrome: '67',
-    firefox: '57',
+    chrome: '61',
```

Comment:
I believe this version of Chrome does not support the full feature set of url search params. I can't remember what is missing but the test suite does cover it

Reply:
Mhm, according to https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#Browser_compatibility it should support it. `.sort` was added in that version

Reply:
However bumping it up to 67 again would be fine.